### ISSUE TITLE
pip install fails on python 3

### ISSUE THREAD

**Issue by pauloyster** [jonathanj/eliottree#64]
```
Requirement already satisfied: jmespath>=0.7.1 in c:\anaconda3\lib\site-packages (from eliot-tree)
Collecting iso8601>=0.1.10 (from eliot-tree)
  Downloading iso8601-0.1.12-py3-none-any.whl
Collecting tree-format>=0.1.1 (from eliot-tree)
  Downloading tree-format-0.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Gal\AppData\Local\Temp\pip-build-0inmxyd1\tree-format\setup.py", line 23, in <module>
        long_description = readme.read()
      File "C:\Anaconda3\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 223: character maps to <undefined>

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\Gal\AppData\Local\Temp\pip-build-0inmxyd1\tree-format\

```

**Comment by jonathanj** [comment]
My suspicion is that this is more of a Windows issue than a Python 3 issue (there are python 3 CI builds but they're for Linux.) Also this appears to be an issue installing `tree-format` which is a dependency and it's likely that the Unicode tree characters in the readme are causing this, which probably affects eliottree in the same way. I'll take a look at improving this.

**Comment by jonathanj** [comment]
@pauloyster I can't reproduce this on a US English build of Windows 10, what version of Windows are you using and what language?

**Comment by pauloyster** [comment]
@jonathanj indeed windows. indeed a tree-format issue (sorry for the confusion).
Windows version is 10.0.16299.19.
Python 3.6.2 :: Anaconda custom (64-bit).

**Comment by pauloyster** [comment]
@jonathanj I think the fix (for both 2 and 3) should be using os.open(filename, encoding='utf-8) during setup (instead of the naiive open(filename)).

**Comment by jonathanj** [comment]
@pauloyster Could you try checkout my tree-format fork, switch to the `10-readme-encoding-setup` and try `python setup.py install` from the checkout root? Possibly even `python setup.py build` might be good enough to expose the issue (you'll need to double check that on master, I'm not 100% sure.)

Unfortunately I can't reproduce this myself on Windows 10.0.15063.

https://github.com/jonathanj/tree-format/tree/10-readme-encoding-setup

**Comment by pauloyster** [comment]
@jonathanj Yes, this installs tree-format (version 0.1.0) properly on my machine. Thanks!

**Comment by jonathanj** [comment]
@pauloyster I've released 0.1.2 of tree-format (and published it on PyPI). Since eliottree's dependency is `tree-format>=0.1.1` that should mean new installs will use 0.1.2. I'll bump the dependency for eliottree in master although I don't have immediate plans for a new release.
